packs/victorops/actions/open_incident.py

```diff
@@ -0,0 +1,17 @@
+import urllib
+from lib.action import VictorOpsAction
+import json
+import time
+
+class OpenIncident(VictorOpsAction):
+  def run(self, severity, entity, message=None):
+    prms = {
+    "message_type": severity,
```

Comment:
Please follow the pep8 style guide - https://www.python.org/dev/peps/pep-0008/

You can run pep8 tool on this file to view the pep8 style violations.